Backpack/Tests/SnapshotTests/BPKFlightLegSnapshotTest.swift

```diff
@@ -47,7 +47,7 @@ final class BPKFlightLegSnapshotTest: XCTestCase {
         duration: String = "7h 55m",
         operatedBy: String? = nil,
         warning: String? = nil,
-        carrierLogo: BPKIconName = .airline
+        carrierLogo: UIImage? = BPKIcon.makeLargeIcon(name: .airline, color: .bpk_black)
```

Comment:
Could we use BPKColor.textOnLight for this? Or (worst case) BPKColor.black? 
We want to reduce the amount of `bpk_` usages.

*Same for the below usages